scripts/build.js

```diff
@@ -50,63 +49,62 @@ if (POSTINSTALL_BUILD_CWD !== CWD) {
       // Fetch package.json
       var pkgJson = require(path.join(CWD, "package.json"));
       var devDeps = pkgJson.devDependencies;
-      // Values listed under `buildDependencies` contain the dependency names
-      // that are required for `lib` building.
-      var buildDependencies = _.pick(devDeps, pkgJson.buildDependencies);
-
-      // Proceed only if there is something to install
-      if (!_.isEmpty(buildDependencies)) {
-        var opts = { env: process.env, stdio: 'inherit' };
+      var opts = { env: process.env, stdio: 'inherit' };
+      var installArgs = [];
 
-        console.log('Building Bookshelf.js')
+      // Map all key (dependency) value (semver) pairs to
+      // "dependency@semver dependency@semver ..." string that can be used
+      // for `npm install` command
+      for (const dep of buildDependencies) {
```

Comment:
Replaced the use of `_.filter` and `_.map` with this `for .. of` cycle, which seems acceptable since Bookshelf is now only supported by node@4 and above, right?